microsoft_student_partners/qpower/organizers.json

```diff
@@ -58,5 +58,13 @@
     "linkedin": "https://www.linkedin.com/in/rprins/",
     "github": "https://github.com/ricardoprins",
     "position": "Event Coordinator"
+  },
+  "Debanjona Bhattacharjya": {
+    "name": "Debanjona Bhattacharjya",
+    "image_url": "https://drive.google.com/file/d/15DYZleWnQU906-8PzTIchWVjBMG_N3PE/view",
```

Comment:
Please use other link like github or linkedin profile pic link, drive link doesn't work here.

Reply:
I have updated the image url. Let me know if it works.